### ISSUE TITLE
License Question

### ISSUE THREAD

**Issue by richardhendricks** [MauveCloud/Ic2ExpReactorPlanner#71]
Hello,
  I see your license is GPL, which is great.  Is all the code written by you?  IE, your reactor simulation code is cleanroom code that emulates the ic2 code without being a copy of it? Interested in re-implementing the reactor via GT to remove some bugs.

**Comment by MauveCloud** [comment]
The code isn't 100% by me - there are a few snippets adapted from code posted on the help site "Stack Overflow" (I'm fairly sure I added comments indicating those), a few things from pull requests by others, and I never did understand the calculations in the exchangers well enough to implement versions of them that I could really call my own, so I got permission to mostly-copy that code.  Presumably Thunderdark (who provided said permission) was aware of the license, and did not ask me to change it, nor did two other IC2 forum admins who apparently also read the message.  I'm not a legal expert, but _hopefully_ that means they'd be okay with others copying/adapting that code from my planner.
